qualtran/bloqs/swap_network/cswap_approx_test.py

```diff
@@ -45,7 +46,9 @@ def test_approx_cswap_t_count(n):
     assert cswap.t_complexity() == cswap_d.t_complexity()
 
 
-def get_t_count_and_clifford(bc: Dict[Bloq, int]) -> Tuple[int, int]:
+def get_t_count_and_clifford(
+    bc: Dict[Bloq, Union[int, sympy.Expr]]
+) -> Tuple[Union[int, sympy.Expr], Union[int, sympy.Expr]]:
```

Comment:
Very optional for this and future PRs, but wanted to bring your attention to `qualtran.resource_counting.symbolic_counting_utils.SymbolicInt` as an alias for `Union[int, sympy.Expr]`. 

Also I might move these symbolic utils to a more easily importable location -- perhaps top-level. But that's todo

Reply:
Good to know.  I will use that in the future.  Yeah, that type is fairly well hidden.